NEWS.md

```diff
@@ -8,6 +8,14 @@
 
 - Fix errors when printing long vertex names (#677, @ahmohamed).
 
+- Fix regression that broke builds on some systems (e.g., GCC version 5 or earlier), introduced in igraph 1.4.0.
+
+## Documentation
+
+- Various improvements (#663, @maelle; #667).
```

Comment:
Isn't it this release that adds vignettes and a pkgdown website? If so, that'd be important to underline.

Reply:
I believe vignettes were added in igraph 1.4.0.